docs/order.rst

```diff
@@ -0,0 +1,142 @@
+            general_fields = ('email', 'first_name', 'last_name', 'watched_issues__name', 'watched_issues__id',
+                              'manual_created_date')
+            direct_serialization_fields = ('created_at', 'email', 'contract', 'solving_issue', 'first_name',
+                                           'last_name', 'manual_created_date')
+            order_fields = ('email', 'solving_issue')
+            extra_order_fields = ('created_at',)
+
+
+    class Issue(models.Model):
+
+        created_at = models.DateTimeField(null=False, blank=False, auto_now_add=True)
+        name = models.CharField(max_length=100, null=False, blank=False)
+        watched_by = models.ManyToManyField('app.User', blank=True, related_name='watched_issues')
+        created_by = models.ForeignKey('app.User', null=False, blank=False, related_name='created_issues')
+        solver = models.OneToOneField('app.User', null=True, blank=True, related_name='solving_issue')
+        leader = models.OneToOneField('app.User', null=False, blank=False, related_name='leading_issue')
+        description = models.TextField(null=True, blank=True)
+
+        class RESTMeta:
+            extra_order_fields = ('solver__created_at',)
+
+
+    class IssueResource(BaseModelResource):
+
+        model = Issue
+        fields = ('id', 'created_at', '_obj_name', 'name', ('created_by', ('id', 'contract', 'created_at')), 'solver',
+                  'leader', 'watched_by')
+        detailed_fields = ('id', 'created_at', '_obj_name', 'name', ('created_by', ('id', 'contract',)), 'solver',
+                           'leader', 'watched_by')
+        general_fields = ('id', '_obj_name', 'name', ('created_by', ('id', 'contract', 'created_at')), 'watched_by')
+        create_obj_permission = True
+        read_obj_permission = True
+        update_obj_permission = True
+        delete_obj_permission = True
+
+
+    class UserResource(BaseModelResource):
+
+        model = User
+        create_obj_permission = True
+        read_obj_permission = True
+        update_obj_permission = True
+        delete_obj_permission = True
+        extra_order_fields = ()
+
+As you can see ``order_fields`` and ``extra_order_fields`` are set inside model RESTMeta for User model. From RESTMeta
+is allowed to filter three fields ('email', 'solving_issue', 'created_at'). But because extra_order_fields is overridden
+inside UserResource client can order only with ('email', 'solving_issue').
+
+Model Issue has set only ``extra_order_fields`` where is allowed to order Issues by User.created_at via related field
+``solver``. Other order fields is generated from all readable fields which is obtained as join of attributes ``fields``,
+``detailed_fields`` and ``general_fields``.
+
+Decorators
+----------
+
+Ordering provides one decorator helper to provide ordering for methods too. Decorator name is ``order_by``. To explain
```

Comment:
Ordering provides one helper to provide ordering for methods as well, decorator ``order_by``. We can use our issue tracker as an example. Issue has a description, but for some reason we can only show the first 50 chars of the description. We can order according this value too, we can define a method ``short_description`` and use the order_by decorator: